Source/Tools/LinuxEmulation/LinuxSyscalls/SignalDelegator.cpp

```diff
@@ -1471,9 +1471,16 @@ namespace FEX::HLE {
       else if (Signal == SIGSEGV &&
                (reinterpret_cast<void*>(ArchHelpers::Context::GetPc(UContext)) == FaultSafeMemcpy::CopyFromUser_FaultLocation ||
                reinterpret_cast<void*>(ArchHelpers::Context::GetPc(UContext)) == FaultSafeMemcpy::CopyToUser_FaultLocation)) {
-        // Jump to the copy fault handler return trampoline to safely return EFAULT.
-        ArchHelpers::Context::SetPc(UContext, reinterpret_cast<uint64_t>(&FaultSafeMemcpy::CopyFaultTrampoline));
-        return;
+        // If you want to emulate EFAULT behaviour then enable this if-statement.
```

Comment:
Why are we making FEX less accurate?

Reply:
@neobrain Is concerned about papering over emulation bugs uncovered through capturing EFAULT correctly. So at least now we can detect it until we find a game/application that requires it.

Reply:
Yeah, OK, fair enough